modules/nodes-cluster-worker-latency-profiles-examining.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * scaling-worker-latency-profiles.adoc (double-check this)
```

Comment:
Was this double checked?

Reply:
@jab-rh Yes! And thank you for seeing that!